translations/ui-receiving/en.json

```diff
@@ -12,7 +12,13 @@
   "title.titleLookUpNoPlugin": "[no instance-selection plugin]",
   "titles.title": "Title",
 
+  "title.modalLabel.edit": "Add title",
```

Comment:
`Edit title`?

Reply:
It's `Add title` in the ticket, let's leave it to Dennis